python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -394,23 +394,24 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
                 unforced = addPreComputedColumns(unforced, fluxToPlotList=self.config.fluxToPlotList,
                                                  toMilli=self.config.toMilli)
                 writeParquet(dataRef_unforced, unforced, badArray=badUnforced)
```

Comment:
If I'm reading this correctly, writeParquet is only called if `doPlotOverlaps` is `True`.  Is that correct?

Reply:
So, no, this is at the same indent level as `doPlotOverlaps`, so is independent.

As for the overlaps needing to go first, by “overlaps” (again, inherited term) we here refer to objects lying on the patch overlap region, so this is a sort of repeatability test since it is based on the same data.  However, we do not want the duplicate measurements in any of the further analyses and these get explicitly removed by selecting against `detect_isPatchInner` for the “bad” arrays (see https://github.com/lsst-dm/pipe_analysis/blob/master/python/lsst/pipe/analysis/utils.py#L974).  While it’s not strictly necessary to have that block fire first, it seemed to follow a natural logic flow to put it ahead of any “bad” assignment.  Note that if 
`writeParquetOnly = True` the `validate` function sets all plotting functions to `False`, including `doPlotOverlaps`, so that block wouldn’t fire if you wanted a “write the parquet files only...do no plotting” run.

Reply:
Okay, the indentation was confusing because the github snippet was poorly chosen.  So it's fine as it is.  Thanks!